CHANGELOG.rst

```diff
@@ -1,6 +1,30 @@
 Changelog
 =========
 
+Unreleased (May 3, 2022)
+-------------------------
+
+Integrating `ancpbids` as the backend for `pybids`. API-breaking changes are as follows,
+where `bl` refers to a `BIDSLayout` object.
+
+The following functions have been removed:
+* `bl.copy_*` — we suggest moving files externally to pyBIDS
+* `bl.build_path`, `bl.write_to_file` — we suggest creating derivatives datasets when adding files via pyBIDS, rather than modifying original datasets
```

Comment:
`build_path` is generally useful when generating derivatives without constructing a new dataset first. I would be curious to see a suggested replacement using ancpbids...

Reply:
Here is an example on how to create derivatives folders:

https://github.com/ANCPLabOldenburg/ancp-bids-apps/blob/main/ancpbidsapps/analysis/nilearn_first_level.py#L66